From 6f70bc1d54887ad3a97dd9c9ae52f86302274932 Mon Sep 17 00:00:00 2001
From: Paul Barrette <paul.barrette@windriver.com>
Date: Tue, 12 Jun 2012 14:24:12 -0400
Subject: [PATCH 24/24] powerpc: uprobes: removed external declaration of
 uprobe_get_swbp_addr.

uprobe_get_swbp_addr is declared as static inline in linux/uprobes.h.
Redeclaring it differently breaks compilation.

Signed-off-by: Paul Barrette <paul.barrette@windriver.com>
---
 arch/powerpc/include/asm/uprobes.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/include/asm/uprobes.h b/arch/powerpc/include/asm/uprobes.h
index bd9e8412..ed6197b 100644
--- a/arch/powerpc/include/asm/uprobes.h
+++ b/arch/powerpc/include/asm/uprobes.h
@@ -40,7 +40,6 @@ struct arch_uprobe_task {
 };
 
 extern int  arch_uprobe_analyze_insn(struct arch_uprobe *aup, struct mm_struct *mm, unsigned long addr);
-extern unsigned long uprobe_get_swbp_addr(struct pt_regs *regs);
 extern int  arch_uprobe_pre_xol(struct arch_uprobe *aup, struct pt_regs *regs);
 extern int  arch_uprobe_post_xol(struct arch_uprobe *aup, struct pt_regs *regs);
 extern bool arch_uprobe_xol_was_trapped(struct task_struct *tsk);
-- 
1.7.5.4

